embedded-tests/src/test/resources/test-data/avro/input_format.json

```diff
@@ -0,0 +1,81 @@
+{
+  "type": "avro_stream",
```

Comment:
Since we are using inline JSON strings / builders for the other stuff, can we also use a builder for this schema definition?